src/PrestaShopBundle/Form/Admin/Sell/Product/Options/ProductAttachmentsType.php

```diff
@@ -0,0 +1,104 @@
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class ProductAttachmentsType extends TranslatorAwareType
+{
+    /**
+     * @var UrlGeneratorInterface
+     */
+    private $urlGenerator;
+
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        UrlGeneratorInterface $urlGenerator
+    ) {
+        parent::__construct($translator, $locales);
+        $this->urlGenerator = $urlGenerator;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('attached_files', EntitySearchInputType::class, [
+                'entry_type' => AttachedFileType::class,
+                'layout' => EntitySearchInputType::TABLE_LAYOUT,
+                'required' => false,
+                'label' => false,
+                'remote_url' => $this->urlGenerator->generate('admin_attachments_search', ['searchPhrase' => '__QUERY__']),
+                'placeholder' => $this->trans('Search file', 'Admin.Catalog.Help'),
+                'empty_state' => $this->trans('No files attached', 'Admin.Catalog.Feature'),
+            ])
+            ->add('add_attachment_btn', IconButtonType::class, [
+                'label' => $this->trans('Add new file', 'Admin.Catalog.Feature'),
+                'icon' => 'add_circle',
+                'type' => 'link',
+                'attr' => [
+                    'data-success-create-message' => sprintf('%s %s',
+                        $this->trans(
+                            'The file was successfully added.',
+                            'Admin.Catalog.Feature'
+                        ),
+                        $this->trans(
+                            'This window will close automatically.',
+                            'Admin.Global'
+                        )
+                    ),
+                    'data-modal-title' => $this->trans('Add new file', 'Admin.Catalog.Feature'),
+                    'class' => 'btn-outline-secondary add-attachment',
+                    'href' => $this->urlGenerator->generate('admin_attachments_create', [
+                        'liteDisplaying' => true,
+                        'saveAndStay' => true,
+                    ]),
+                ],
+            ])
+        ;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+        $resolver->setDefaults([
+            'label' => $this->trans('Attached files', 'Admin.Catalog.Feature'),
+            'label_tag_name' => 'h2',
+            'label_help_box' => $this->trans('Instruction manuals, technical works or any file you want to add to a product.', 'Admin.Catalog.Help'),
```

Comment:
```suggestion
            'label_help_box' => $this->trans('Instructions, size guide, or any file you want to add to a product.', 'Admin.Catalog.Help'),
```
Sorry, the wording changed after I discussed it with Mateus and Tristan.

Reply:
The help text below should be: "Customers can download these files on the product page."

Reply:
![image](https://user-images.githubusercontent.com/70583503/130211976-6d5de5b5-b55f-408f-a472-a2498fd34ac5.png)

Reply:
Ok! Done ^^